mockgcp/mockstorage/bucket.go

```diff
@@ -113,19 +127,30 @@ func (s *buckets) InsertBucket(ctx context.Context, req *pb.InsertBucketRequest)
 	now := timestamppb.Now()
 
 	obj := proto.Clone(req.GetBucket()).(*pb.Bucket)
-	obj.Id = PtrTo(fqn)
+	obj.Id = PtrTo(name.Bucket)
 	obj.Kind = PtrTo("storage#bucket")
-	obj.Name = PtrTo(fqn)
+	obj.Name = PtrTo(name.Bucket)
 	obj.ProjectNumber = PtrTo(uint64(project.Number))
+
 	obj.Location = PtrTo("US")
 	obj.LocationType = PtrTo("multi-region")
 	obj.Rpo = PtrTo("DEFAULT")
-	obj.SelfLink = PtrTo(fmt.Sprintf("https://www.googleapis.com/storage/v1/b/%s", obj.GetName()))
+	obj.SelfLink = PtrTo(fmt.Sprintf("https://www.googleapis.com/storage/v1/b/%s", name.Bucket))
 	obj.StorageClass = PtrTo("STANDARD")
 	obj.TimeCreated = now
 	obj.Updated = now
 	obj.Metageneration = PtrTo(int64(1))
 
+	obj.Generation = PtrTo(time.Now().UnixNano())
+
+	switch req.GetBucket().GetLocation() {
+	case "us-central1":
+		obj.Location = PtrTo("US-CENTRAL1")
+		obj.LocationType = PtrTo("region")
+		obj.Rpo = nil
+		obj.SatisfiesPZI = PtrTo(true)
+	}
```

Comment:
Can we extend this to support some more location value and type?  Or have some way to make it more discoverable that StorageBucket can only have `us-central1` if not using the default multi-region US.